components/builder-protocol/protocols/sessionsrv.proto

```diff
@@ -36,6 +36,36 @@ message AccountUpdate {
   optional string email = 2;
 }
 
+message AccessToken {
+    optional uint64 account_id = 1;
+    optional uint32 flags = 2;
+    optional int64 expires = 3;
```

Comment:
Would this make more sense as a `uint64`? We don't have negative timestamps, do we?

Reply:
Yeah.. it should ideally be u64.. however chrono timestamp() returns i64, and postgres natively stores i64, so might as well leave it i64 and not do conversions all over the place.

Reply:
On second thought, `chrono` supports it for the sake of representing dates before the epoch, so maybe we should keep it.